easybuild/easyconfigs/s/subunit/subunit-1.4.5-GCCcore-13.3.0.eb

```
@@ -0,0 +1,59 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+easyblock = 'ConfigureMake'
+
+name = 'subunit'
+version = '1.4.5'
+
+homepage = "https://github.com/testing-cabal/subunit"
+description = """Subunit is a streaming protocol for test results."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/testing-cabal/subunit/archive/refs/tags/']
+sources = ['%(version)s.tar.gz']
+checksums = ['c55b39d0802ca15c18fbf267faf3a1cf67b4fe2482abbb00755c31b53c833bf7']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('Autotools', '20231222'),
+    ('pkgconf', '2.2.0'),
+    ('poetry', '1.8.3'),
+]
+
+dependencies = [
+    ('Check', '0.15.2'),
+    ('CppUnit', '1.15.1'),
+    ('Python', '3.12.3'),
+]
+
+preconfigopts = "autoreconf -vi &&"
+
+exts_defaultclass = 'PythonPackage'
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+}
+
+exts_list = [
+    ('iso8601', '2.1.0', {
+        'checksums': ['6b1d3829ee8921c4301998c909f7829fa9ed3cbdac0d3b16af2d743aed1ba8df'],
+    }),
+    ('pbr', '6.1.0', {
+        'checksums': ['788183e382e3d1d7707db08978239965e8b9e4e5ed42669bf4758186734d5f24'],
+    }),
+    ('fixtures', '4.1.0', {
+        'checksums': ['82b1c5e69f615526ef6c067188a1e6c6067df7f88332509c99f8b8fdbb9776f3'],
+    }),
+    ('testtools', '2.7.2', {
+        'checksums': ['5be5bbc1f0fa0f8b60aca6ceec07845d41d0c475cf445bfadb4d2c45ec397ea3'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['lib/libsubunit.a', 'lib/libsubunit.%s' % SHLIB_EXT],
+    'dirs': ['include/subunit'],
+}
+
+sanity_check_commands = ['python -c "import subunit"']
```

```suggestion
sanity_check_commands = ['python -s -c "import subunit"']
```
tests are complaining about the missing `-s`:
```
AssertionError: Python invocation in 'sanity_check_commands = ['python -c "import subunit"']' (line #57) should use the '-s' parameter in subunit-1.4.5-GCCcore-13.3.0.eb
```